packages/core/src/services/evaluations/run.ts

```diff
@@ -102,50 +64,16 @@ export const runEvaluation = async (
     },
   })
 
-  return Result.ok({
-    ...run,
-    response: run.response.then(async (res) => {
-      await handleEvaluationResponse(
-        res as ChainStepResponse<'object'> | undefined,
-        documentUuid,
-        evaluation,
-        documentLog,
-      )
-
-      return res
-    }),
+  const responseResult = (await run.response) as ChainResponse<'object'>
+  const processedResponse = await handleEvaluationResponse({
+    errorableUuid: run.errorableUuid,
+    responseResult,
+    documentUuid,
+    evaluation,
+    documentLog,
   })
-}
 
-async function handleEvaluationResponse(
-  response: ChainStepResponse<'object'> | undefined,
-  documentUuid: string,
-  evaluation: EvaluationDto,
-  documentLog: DocumentLog,
-) {
-  if (!response) return
-  if (response.object === undefined) {
-    throw new LatitudeError(
```

Comment:
Evaluation run never throws errors now. If it throws an error is a bug on our code

Reply:
Why? Could an evaluation not contain errors in their syntax logic, or with the provider?

Reply:
We were checking here if the response from the AI had `object` key in the response. This is still checked but in another place.